tools/bioimaging/bioimage_inference.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="bioimage_inference" name="Load model from BioImage and make inferences" version="0.0.5">
+    <description>using images</description>
+    <requirements>
+        <requirement type="package" version="3.9.12">python</requirement>
+        <requirement type="package" version="2.1.2">pytorch</requirement>
+        <requirement type="package" version="0.16.1">torchvision</requirement>
+        <requirement type="package" version="3.0.8">cython</requirement>
+    </requirements>
+    <version_command>echo "@VERSION@"</version_command>
+    <command detect_errors="aggressive">
+<![CDATA[
+        python '$__tool_directory__/main.py'
+
+            --imaging_model '$input_imaging_model'
+            --image_file '$input_image_file'
+]]>
+    </command>
+    <inputs>
+        <param name="input_imaging_model" type="data" format="zip" label="BioImage model" help="Please load a BioImage model from file uploader"/>
+        <param name="input_image_file" type="data" format="npz" label="Image to be analysed" help="Please provide an image"/>
```

Comment:
NPZ is numpy isn't it? How do people create such image? Should we create an NPZ file from a tiff, png as part of this tool?

Reply:
chiming in on this on @beatrizserrano 's request:
(disclaimer: I'm unaware of the context of this PR, but can report from the bioimage.io side: We demand `.npy` test files to be provided alongside every model contributed to bioimage.io. Among the utility functions we provide in bioimageio.core there is
https://github.com/bioimage-io/core-bioimage-io-python/blob/f798344213c179a6c836938aff9e4f6c46f8d23c/bioimageio/core/utils/image_helper.py#L136
 -- a util function using imageio to attempt to load any given image and then attempting to guess it's axes, see 
 https://github.com/bioimage-io/core-bioimage-io-python/blob/f798344213c179a6c836938aff9e4f6c46f8d23c/bioimageio/core/utils/image_helper.py#L42
 
Software using the model zoo (or even specifically bioimageio.core) should try to delegate this guesswork to the end user instead.

note: the links above are from a development branch. These updated functions will be available in the coming bioimageio.core release, currently this functionality is implemented [here](https://github.com/bioimage-io/core-bioimage-io-python/blob/main/bioimageio/core/image_helper.py)

Reply:
Would generating the .npy files/objects at validation time using the spec be sensible/viable?